x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -341,32 +348,50 @@ export function MlLatencyCorrelations({ onClose }: Props) {
         </>
       ) : null}
 
-      {histograms.length > 0 && selectedHistogram !== undefined && (
-        <CorrelationsTable
-          // @ts-ignore correlations don't have the same column format other tables have
-          columns={mlCorrelationColumns}
-          // @ts-expect-error correlations don't have the same significant term other tables have
-          significantTerms={histogramTerms}
-          status={FETCH_STATUS.SUCCESS}
-          setSelectedSignificantTerm={setSelectedSignificantTerm}
-          selectedTerm={{
-            fieldName: selectedHistogram.field,
-            fieldValue: selectedHistogram.value,
-          }}
-          onFilter={onClose}
-        />
+      <div data-test-subj="apmCorrelationsTable">
+        {histograms.length > 0 && selectedHistogram !== undefined && (
+          <CorrelationsTable
+            // @ts-ignore correlations don't have the same column format other tables have
+            columns={mlCorrelationColumns}
+            // @ts-expect-error correlations don't have the same significant term other tables have
+            significantTerms={histogramTerms}
+            status={FETCH_STATUS.SUCCESS}
+            setSelectedSignificantTerm={setSelectedSignificantTerm}
+            selectedTerm={{
+              fieldName: selectedHistogram.field,
+              fieldValue: selectedHistogram.value,
+            }}
+            onFilter={onClose}
+          />
+        )}
+        {histograms.length < 1 && progress > 0.99 ? (
+          <>
+            <EuiSpacer size="m" />
+            <EuiText textAlign="center">
+              <FormattedMessage
+                id="xpack.apm.correlations.latencyCorrelations.noCorrelationsText"
+                defaultMessage="No significant correlations found"
+              />
+            </EuiText>
+          </>
+        ) : null}
+      </div>
+      {log.length > 0 && displayLog && (
+        <EuiAccordion id="accordion1" buttonContent="Log">
```

Comment:
Does `Log` here needs internationalization?

Reply:
Fixed translation in 7592578.